tests/unit/unittests.supp

```diff
@@ -0,0 +1,8 @@
+{
+   <Cloned SIValue strings>
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:malloc
+   fun:strdup
+   fun:SI_StringVal
+}
```

Comment:
I added this suppression because in a lot of tests, we construct elements like:
`AR_ExpNode *str = AR_EXP_NewConstOperandNode(SI_StringVal("muchacho"));`
Which is convenient for testing, but not an accurate reflection of how we build `AR_ExpNode`s in the module. We can instead try to be more stringent about this, but I don't think a suppression is a terrible choice for now.